boltdesignsystem/bolt
# DS-650 PatternLab archive folder

## Jira

https://pegadigitalit.atlassian.net/browse/DS-650

## Summary

An archive folder was created

## Details

The following directories were transferred from `50-pages` folder into the new `archive` folder.

- .../05-d8-homepage
- .../10-d8-product-pages
- .../15-d8-resources-details-pages
- .../20-d8-press-and-media
- .../25-d8-agenda-manager
- .../30-d8-partners
- .../35-d8-events
- .../40-d8-content-hub
- .../45-d8-blog
- .../50-d8-careers
- .../55-d8-calculator

## How to test

Pull the branch. 

- Check if all required directories were moved into a new archive folder.
- The Archive folder and Tests folder should be able to toggle with a left nav button.
